doc/ir-semantics.text

```diff
@@ -106,11 +103,11 @@ Update (partial) function, returns an H:
   (P, C*) : (I, T, H, E, l) --> (P, C*) : step I(l) (P, I, T, H, E, l)
 
   (P, C*) : (I, T, H, E, l) --> (P, (C*, C')) : (I', T, H, E', start)
-    when I(l) = (const x' = call f (e, ...))
+    when I(l) = (call x = f e*)
```

Comment:
typo: `x'`, since it goes into continuation (last line) and should not collide with the xs from the function declaration